Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardInputViewModel.swift

```diff
@@ -138,10 +161,11 @@ class CreditCardInputViewModel: ObservableObject {
         return RemoveCardButton.AlertDetails(
             alertTitle: Text(CreditCardText.RemoveCardTitle),
             alertBody: Text(CreditCardText.RemoveCardSublabel),
-            primaryButtonStyleAndText: .destructive(Text(CreditCardText.RemovedCardLabel)) { [self] in
-                guard let creditCard = creditCard else { return }
-
-                removeSelectedCreditCard(creditCardGUID: creditCard.guid)
+            primaryButtonStyleAndText: .destructive(Text(CreditCardText.RemovedCardLabel)) {
+                [self] in
```

Comment:
Just noticed this `[self] in`. If we just use the normal self without the weakify, do we really need to put the `[self]` ? I normally just put `_ in` or nothing if there's no returned param (in the case again we don't want to weakify).

Reply:
Actually, it depends and in this case I missed `[unowned self]` to be added as part of the capture list during the refactor